src/context/ArchiveContext/ArchiveContext.jsx

```diff
@@ -0,0 +1,71 @@
+import { useContext, createContext, useState, useEffect } from 'react';
+import { useAxios } from '../../CustomHooks';
+
+const ArchiveContext = createContext();
+
+const ArchiveProvider = ({ children }) => {
+	const [archiveNotes, setArchiveNotes] = useState([]);
+	const { response: archiveResponse, operation: fetchArchive } = useAxios();
+
+	useEffect(() => {
```

Comment:
Appreciation: I really like the thinking behind the effect hook here and how any api call will trigger the effect as response is passed as a dependency.